packages/effect/src/Schema.ts

```diff
@@ -6311,6 +6311,22 @@ export class DateTimeUtcFromNumber extends transformOrFail(
   }
 ).annotations({ identifier: "DateTimeUtcFromNumber" }) {}
 
+/**
+ * Defines a schema that attempts to convert a valid `Date` to a `DateTime.Utc` instance using the `DateTime.unsafeMake` constructor.
+ *
+ * @category DateTime.Utc transformations
+ * @since 3.11.0
+ */
+export class DateTimeUtcFromDate extends transformOrFail(
+  ValidDateFromSelf.annotations({ description: "a valid Date that will be parsed into a DateTime.Utc" }),
```

Comment:
You can use DateFromSelf here as `decodeDateTime` will validate the Date instance.

Reply:
done